frontend/src/features/workspace/components/CreateFormModal/CreateFormModalContent/FormResponseOptions.tsx

```diff
@@ -89,21 +93,25 @@ export const FormResponseOptions = forwardRef<
           <Tile.Subtitle>{mrf.subtitle}</Tile.Subtitle>
           <OptionDescription
             listItems={[
-              { text: 'Supports approval workflows', badge: 'New' },
-              { text: 'Up to Restricted and Sensitive (Normal) data' },
+              { text: 'Supports email submissions' },
+              { text: 'Supports approval workflows' },
+              {
+                text: 'Up to Confidential (Cloud-Eligible) and Sensitive (High) data',
+                badge: 'new',
+              },
             ]}
           />
         </Tile>
       </Stack>
       {/* TODO: (Kill Email Mode) Remove this route after kill email mode is fully implemented. */}
-      <InlineMessage mt="1rem">
+      {/* <InlineMessage mt="1rem">
```

Comment:
Suggestion (code cleanliness): Shall we remove this instead of commenting it out? Since we can still refer to the past in git version history